extension/data.ts

```diff
@@ -86,6 +86,27 @@ interface DeinflectionRuleGroup {
   rules: DeinflectionRule[];
 }
 
+const WORD_DICT_ENTRY_REGEX = /^(.+?)\s+(?:\[(.*?)\])?\s*\/(.+)\//;
+
+type WordDictEntry = {
+  word: string;
+  pronunciation: string | null;
```

Comment:
In the interest of avoiding null in types I wonder if it's possible to just make this `string` type and use empty string if there's no pronunciation? That tends to make things easier to reason about and allows avoiding guards.

Of course, if we always need to check for empty string instead, then there's not much difference in which case keeping it like this is maybe more consistent.

Reply:
Maybe the only benefit to using `null` here would be that it forces you to explicitly set a default value or explicitly handle empty pronunciation values. This isn't a huge benefit. I could also see that also as a negative depending on the situation - it might be annoying to have to explicitly set a default value, so I'm fine with just setting this as an empty string instead of using `null`.